ngraph/core/reference/include/ngraph/runtime/reference/roi_pooling.hpp

```diff
@@ -140,11 +140,15 @@ namespace ngraph
                                 {
                                     T in_y =
                                         (pooled_h > 1)
-                                            ? (ph * roi_height_scale + roi_h_start * (height - 1))
+                                            ? ((ph == pooled_h - 1) ? (height - 1) * roi_h_end
```

Comment:
Same here, would be good to have a comment about the corner case.

Reply:
Done